common/app/common/configuration.scala

```diff
@@ -323,7 +323,7 @@ class GuardianConfiguration extends GuLogging {
     lazy val isDefined: Boolean = hostOption.isDefined && portOption.isDefined
 
     private lazy val hostOption = Option(System.getenv("proxy_host"))
-    private lazy val portOption = Option(System.getenv("proxy_port")) flatMap { _.toIntOption }
+    private lazy val portOption = Option(System.getenv("proxy_port")) flatMap { augmentString(_).toIntOption }
```

Comment:
Here we are trying to call `toIntOption`, a method of `StringOps` on a `String` so we're getting the error:

```
value toIntOption is not a member of String
```

Normally, we would expect [implicit conversions](https://www.oreilly.com/library/view/scala-cookbook/9781449340292/ch01s01.html) to have worked and change `String` to `StringOps` automatically but in this case they don't because the compiler gets confused:

```
[error] Note that implicit conversions are not applicable because they are ambiguous:
[error]  both method augmentString in object Predef of type (x: String): scala.collection.StringOps
[error]  and method String2ToOptions in trait Strings of type (s: String): common.package.String2ToOptions
[error]  are possible conversion functions from x$4.type to ?{def toIntOption: ?}
```

So we have to explicitly call `augmentString()` to turn `String` to `StringOps` and have access to `toIntOption` method.